quickwit/quickwit-cli/tests/cli.rs

```diff
@@ -1073,3 +1078,85 @@ async fn test_cmd_all_with_s3_localstack_internal_api() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_index_stats() -> anyhow::Result<()> {
+    let index_id = append_random_suffix("index-stats-env");
```

Comment:
You don't have to randomize those if they're not going to be written to a shared test resource (LocalStack, Kafka, ...)